smac_planner/src/a_star.cpp

```diff
@@ -365,6 +384,84 @@ bool AStarAlgorithm<NodeT>::isGoal(NodePtr & node)
 }
 
 template<>
+AStarAlgorithm<NodeSE2>::NodePtr AStarAlgorithm<NodeSE2>::getAnalyticPath(const NodePtr & node, const ValidityChecker & checker)
```

Comment:
I think `checker` is the wrong word (along with `ValidityChecker` since its really just getting neighbors from the graph). I think these should be renamed

Reply:
I was trying to think a way to make this function more easy to understand, but I think there's not really a great way that wouldn't involve a bunch more copies. Lets leave as is.

Reply:
Renamed for clarity.